Lecture-4/Task3/Task3/Application.java

```diff
@@ -0,0 +1,50 @@
+package Task3;
+
+import java.util.Scanner;
+
+public class Application {
+
+    private static int rhombSide;
+    private static StringBuffer stringBuffer1;
+    private static StringBuffer stringBuffer2;
```

Comment:
also not descriptive `StringBuffer ` names

Reply:
done